db/migrate/20230704182448_create_flipper_tables.rb

```diff
@@ -0,0 +1,22 @@
+class CreateFlipperTables < ActiveRecord::Migration[7.0]
+  def self.up
+    create_table :flipper_features do |t|
+      t.string :key, null: false
+      t.timestamps null: false
+    end
+    add_index :flipper_features, :key, unique: true
+
+    create_table :flipper_gates do |t|
+      t.string :feature_key, null: false
+      t.string :key, null: false
+      t.string :value
+      t.timestamps null: false
+    end
+    add_index :flipper_gates, %i[feature_key key value], unique: true
+  end
+
+  def self.down
```

Comment:
Should we use reversible migrations? Just asking.

Reply:
this is being auto-generated by the flipper installation process, not sure if we should change it